app/models/todo.rb

```diff
@@ -0,0 +1,2 @@
+class Todo < ActiveRecord::Base
```

Comment:
Consider adding a presence validation on title, possibly others -- do we want todos with blank information?